packages/core/src/Serwist.ts

```diff
@@ -101,6 +103,13 @@ export interface SerwistOptions {
    * @default false
    */
   disableDevLogs?: boolean;
+  /**
+   * Static route rules that define how certain resources should be fetched
+   * even before the service worker starts up. 
+   * 
+   * @see https://developer.mozilla.org/en-US/docs/Web/API/InstallEvent/addRoutes
+   */
+  requestRouteRules?: RequestRouteRule | RequestRouteRule[];
```

Comment:
I think `requestRules` is better than `requestRouteRules` to avoid confusion between Serwist's routes and these routes?

Reply:
renamed